lib/renderer/api/ipc-renderer.ts

```diff
@@ -23,15 +21,9 @@ ipcRenderer.sendTo = function (webContentsId, channel, ...args) {
   return ipc.sendTo(internal, false, webContentsId, channel, args)
 }
 
-ipcRenderer.sendToAll = function (webContentsId, channel, ...args) {
```

Comment:
I'm pretty sure we use this for our chrome extension stuff.

Reply:
we use `ipcRendererInternal.sendToAll()` there

Reply:
https://github.com/electron/electron/blob/master/typings/internal-electron.d.ts#L65-L67

Reply:
this is about the public API

Reply:
@MarshallOfSound do you think the public API should be documented instead being removed as undocumented?